go.mod

```diff
@@ -98,7 +98,7 @@ require (
 	github.com/mitchellh/go-ps v0.0.0-20190716172923-621e5597135b
 	github.com/mitchellh/go-testing-interface v1.0.3
 	github.com/mitchellh/hashstructure v1.0.0
-	github.com/mitchellh/mapstructure v1.3.1
+	github.com/mitchellh/mapstructure v1.3.3
```

Comment:
maybe this is from something else on local? I think go.mod and go.sum should be able to be restored to whatever is on HEAD